icu4c/source/common/uloc.cpp

```diff
@@ -1192,23 +1194,23 @@ ulocimp_getLanguage(const char *localeID,
         }
     }
 
-    if(pEnd!=nullptr) {
-        *pEnd=localeID;
+    if (sink != nullptr && !result.isEmpty()) {
+        sink->Append(result.data(), result.length());
```

Comment:
nit: Append(result.toStringPiece())  ?

Reply:
I'll soon replace it with writing directly to the sink instead.